### PR TITLE
new package: qt5-qtwebengine

### LINKED ISSUE #10070: [Package]: qtwebengine
### Package description

The Qt WebEngine module provides a web browser engine that makes it easy to embed content from the World Wide Web into your Qt application on platforms that do not have a native web engine.

Qt WebEngine provides C++ classes and QML types for rendering HTML, XHTML, and SVG documents, styled using Cascading Style Sheets (CSS) and scripted with JavaScript. HTML documents can be made fully editable by the user through the use of the contenteditable attribute on HTML elements.

### Home page URL

_No response_

### Source code URL

https://github.com/qt/qtwebengine/tree/v5.15.9-lts

### Packaging policy acknowledgement

- [X] I have read and understand the [Packaging Policy](https://github.com/termux/x11-packages/blob/master/CONTRIBUTING.md#packaging-policy).

### Additional information

To get better html5 compatibility for modern web pages.
It's a significant package that will make revolution of the termux-x11 environment

### PR DESCRIPTION
**Pull request by licy183**
Closes #10070

### CODE REVIEW

**Comment by xtkoba** [comment]
Great work.

I suggest `qt5-qtwebengine` as the package name, aligning with other `qt5-qt*` packages.

**Comment by licy183** [comment]
Emmm... I just noticed that some hacking method is used to install packages in the builder... Will fix it tomorrow.

File: x11-packages/qt5-qtwebengine/build.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/qt/qtwebengine
+TERMUX_PKG_DESCRIPTION="Qt 5 Web Engine Library"
+TERMUX_PKG_LICENSE="LGPL-3.0, LGPL-2.1, BSD 3-Clause"
+TERMUX_PKG_MAINTAINER="@termux"
```
Comment (xtkoba) [inline x11-packages/qt5-qtwebengine/build.sh:4]: Who maintain(s) this elephantine package?

Comment (licy183) [reply x11-packages/qt5-qtwebengine/build.sh:4]: Should I set it to myself?

Comment (licy183) [reply x11-packages/qt5-qtwebengine/build.sh]: Set maintainer to me now.

**Pull request merged**